### PR TITLE
Fix DS settings issues on IE and Edge

### PR DESCRIPTION
**Pull request by huffman**
https://highfidelity.fogbugz.com/f/cases/13337/IE-non-edge-fails-to-list-content-archives

### CODE REVIEW

**Comment by birarda** [comment]
#### Test Plan

1. Run the Sandbox from this PR. With IE11 and a non IE browser (chrome/firefox) confirm that the table of content archives loads on the 'Content' page in the domain-server web interface.
2. Using IE Edge and a non IE browser, upload the attached models.json.gz file. Confirm that the upload completes and the restart modal appears to show the the domain server is restarting to apply the change.

[castle_Aug15.json.gz](https://github.com/highfidelity/hifi/files/1820849/castle_Aug15.json.gz)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8114/app_12673_PR.apk

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8114/HighFidelity-Beta-PR12673-debea3791de4d9aa6ac22b329c705e1bcc365063.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8114/HighFidelity-Beta-PR12673-debea3791de4d9aa6ac22b329c705e1bcc365063.exe

File: libraries/embedded-webserver/src/HTTPConnection.cpp [hunk 1]
```diff
@@ -251,7 +251,7 @@ void HTTPConnection::readHeaders() {
         if (trimmed.isEmpty()) {
             _socket->disconnect(this, SLOT(readHeaders()));
 
-            QByteArray clength = _requestHeaders.value("Content-Length");
+            QByteArray clength = _requestHeaders.value("content-length");
```
Comment (Atlante45) [inline libraries/embedded-webserver/src/HTTPConnection.cpp]: Maybe use `requestHeader` here too?

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Files upload and content is formatted correctly on all browsers

**Comment by conklin94122** [comment]
Retest after changes.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8118/app_12673_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8121/app_12673_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8118/HighFidelity-Beta-PR12673-e703d173715937db9419cd1670ab32c9b93f2469.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8118/HighFidelity-Beta-PR12673-e703d173715937db9419cd1670ab32c9b93f2469.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8121/HighFidelity-Beta-PR12673-fdca6e083253d88db1c09c3c45c31671bfbb6035.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8121/HighFidelity-Beta-PR12673-fdca6e083253d88db1c09c3c45c31671bfbb6035.dmg

**Comment by L370** [comment]
Just to clarify, after uploading the file, you should see the server quit and restart (on it's own, with no prompting)

**Comment by L370** [comment]
@conklin94122 

Tests verified in IE, Firefox and Chrome.

Please note that initial test in Chrome resulted in the Webpage not refreshing correctly (blank page with only the navigation bar being present).  3 subsequent tests in Chrome were successful with no issues.

Tested in IE - tests passed

Tested in Firefox - tests passed

**Pull request merged**